index.js

```diff
@@ -0,0 +1,169 @@
+alert("Welcome to calculator");
+
+var typeCalculator = prompt("Choose (s)imple / (a)dvanced / (b)mi / (t)rip");
+console.log(typeCalculator);
```

Comment:
Don't leave console.logs in the code you're going to push to master - consider this your final product.